readme.txt

```diff
@@ -64,6 +69,18 @@ and pull requests are welcome!
 
 Install from the WordPress plugin directory or put `micropub.php` in your plugin directory. No setup needed.
 
+== Configuration Options ==
+
+These configuration options can be enabled by adding them to your wp-config.php
+
+*  define('MICROPUB_LOCAL_AUTH', '1') - Bypasses Micropub authentication in 
```

Comment:
very minor style nits: backticks around the code snippets please, and only one space here after `*`